### PR TITLE
Install: clarifictions around Python library methods

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

File: content/docs/install/linux.md [hunk 1]
```diff
@@ -1,17 +1,29 @@
 # Installation on Linux
 
-> To use DVC [as a Python library](/doc/api-reference), please
-> [install with pip](#install-with-pip) or [with conda](#install-with-conda).
+<admon type="tip">
+
+To use DVC [as a Python library](/doc/api-reference), please install
+[with `pip`](#install-with-pip) or [with `conda`](#install-with-conda).
```
Comment (jorgeorpinel) [inline content/docs/install/linux.md:6]: What about Snap? I'm not familiar.

Comment (shcheklein) [reply content/docs/install/linux.md:6]: no, it can't be use as a lib ...

File: content/docs/install/linux.md [hunk 2]
```diff
@@ -1,17 +1,29 @@
 # Installation on Linux
 
-> To use DVC [as a Python library](/doc/api-reference), please
-> [install with pip](#install-with-pip) or [with conda](#install-with-conda).
+<admon type="tip">
+
+To use DVC [as a Python library](/doc/api-reference), please install
```
Comment (shcheklein) [inline content/docs/install/linux.md:5]: should we have the same note in the mac os?

**Review by shcheklein — APPROVED** [review]
one comment, otherwise LGTM

File: content/docs/install/linux.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/install/linux.md:5]: It's there 🙂 See https://dvc-org-install-py-lib-luyi0e0.herokuapp.com/doc/install/macos

**Pull request merged**